docs/core/tools/dotnet-nuget-add-source.md

```diff
@@ -3,7 +3,7 @@ title: dotnet nuget add source command
 description: The dotnet nuget add source command adds a new package source to your NuGet configuration files. 
 ms.date: 03/20/2020
 ---
-# dotnet nuget add source
+# `dotnet nuget add source`
```

Comment:
I recently saw these `blocks` removed from headers in another PR, as if we're not supposed to have them.

Reply:
Right, we don't apply styles to headings.  https://docs.microsoft.com/en-us/contribute/text-formatting-guidelines#headings-and-link-text